build.gradle.kts

```diff
@@ -36,18 +36,41 @@ semver {
     // Do not create an empty release commit when running the "releaseVersion" task.
     createReleaseCommit = false
 
+    // Use "RC" instead of "SNAPSHOT" as the default pre-release identifier.
+    defaultPreRelease = "RC"
+
     // Do not let untracked files bump the version or add a "-SNAPSHOT" suffix.
     noDirtyCheck = true
 }
 
 // Only override a default version (which usually is "unspecified"), but not a custom version.
 if (version == Project.DEFAULT_VERSION) {
-    version = semver.semVersion.takeIf { it.isPreRelease }
-        // To get rid of a build part's "+" prefix because Docker tags do not support it, use only the original "build"
-        // part as the "pre-release" part.
-        ?.toString()?.replace("${semver.defaultPreRelease}+", "")
-        // Fall back to a plain version without pre-release or build parts.
-        ?: semver.version
+    val semVersion = semver.semVersion
+
+    version = buildString {
+        // Use the major, minor, and patch version as the version string, for example, "1.2.3".
+        append(semVersion.major)
+        append(".")
+        append(semVersion.minor)
+        append(".")
+        append(semVersion.patch)
```

Comment:
FYI, this should be the same as `append(semVersion.toVersionString(v2 = false))`.